wa/output_processors/postgresql.py

```diff
@@ -0,0 +1,478 @@
+#    Copyright 2018 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import uuid
+import collections
+
+try:
+    import psycopg2
+    from psycopg2 import (connect, extras)
+    from psycopg2 import Error as Psycopg2Error
+except ImportError as e:
+    psycopg2 = None
+    import_error_msg = e.args[0] if e.args else str(e)
+
+from wa.utils import postgres_convert
+from wa import OutputProcessor, Parameter, OutputProcessorError
+from wa.utils.types import level
+from wa.framework.target.info import CpuInfo
+
+from devlib.target import KernelVersion, KernelConfig  # pylint: disable=wrong-import-order
+
+
+class PostgresqlResultProcessor(OutputProcessor):
+
+    name = 'postgres'
+    description = """
+    Stores results in a Postgresql database.
+
+    This can also be easily ascertained from the postgres_schema.sql file:
+    ../commands/postgres_schema.sql
```

Comment:
Don't need `also` here anymore, and I think the aim was to provide the full path of the file rather than the relative path as not all users will know the location of the installation. This can then be dynamically populated when using the show command etc.

Reply:
Done